en/modules/admin/pages/initiatives.adoc

```diff
@@ -1,14 +1,369 @@
+
+== 0. Initiative setup
+
+How initiatives work depends in two things: how the initiative module is configured through the xref:admin:initiatives_initializer.adoc[initializer] (in code) and how the initiative type is configured in the administration panel.
+
+Regarding initiatives type, **an administrator** need to define one. There must be at least one type for initiatives to work. If there's only one then the participant will not get the possibility to choose the initiative type.
+
+Some real examples from the city of Barcelona are:
+
+* To add an item to the Municipal Council or Districts agendas
+* To perform a participatory process
+* To create a new assembly
+* To approve or modify a regulation
+* To request the holding of a citizen consultation
+* To call for assemblies (neighborhood councils or public audiences)
+
+=== New initiative type form
+
+image:admin_initiative_type.png[New initiative type form]
+
+.Admin new initiative type form: General information
+|===
+|Field |Type |Description
+
+|Title
+|Required
+|Title of the initiative type. For instance, "New park at city center".
+
+|Description
+|Required
+|Explanation of what can be done with this initiative type.
+
+|Banner image
+|Required
+|An image for this initiative type. NOTE: At the moment this is not used.
+
+|===
+
+.Admin new initiative type form: Options
+|===
+|Field |Type |Description
+
+|Signature type
+|Optional
+|Which kind of signature type will be allowed. Options are: Online, In-person and Mixed. The Mixed option allows a participant to choose which signature type will have the initiative.
+
+|Enable attachments
+|Optional
+|Wheter it'll allow to upload attachments to initiatives.
+
+|Enable participants to undo their online signatures
+|Optional
+|Can a participant undo the signature once it have already signed?
+
+|Enable authors to choose the end of the signature collection period
+|Optional
+|Can the authors of an initiative change the end of the signature collection period?
+
+|Enable authors to choose the area for their initiative
+|Optional
+|Can the authors choose the area of their initiative?
+
+|Enable child scope signatures
+|Optional
+|This config flag doesn't support offline votes, it enables sub-scopes and works with an authorization handler that associates a scope to the user, make sure you select that authorization, bellow in authorization settings. For it to work, scopes need to be configured in a hierarchical way : 1 Parent - N Child.
+
+|Only allow global scope initiatives creation
+|Optional
+|Tick this flag if you enabled "Child scope signature" and configured the global scope as your parent scope. By enabling this, initiative type selection will be skipped in the initiative creation wizard.
+
+|Enable promoting committee
+|Optional
+|Wheter if it will allow individual initiatives (made by only one person) or it will always have to have a collective (multiple promoters, being multiple individuals or a user group).
```

Comment:
```suggestion
|Whether it will only allow individual initiatives (made by only one person) or it will always have to have a collective (multiple promoters, being multiple individuals or a user group).
```